nilearn/glm/tests/test_dmtx.py

```diff
@@ -473,13 +314,11 @@ def test_design_matrix17():
     assert (X[ct, 0] > 0).all()
 
 
-def test_design_matrix18():
+def test_design_matrix_scaling_blocks(frame_times):
```

Comment:
Would it be possile to factor this with the previous test ?

Reply:
yup 
done